master-MC1.12 fails to build, cant find Extra Cells API

Tried building from master-MC1.12, looks like the link for the Extra Cells API used for compilation is gone. The xml isn't being served at the current link.

The url is defined in **build.gradle line 83** and the extra cells version is defined in **build.properties line 12**.

You can see the output from `./gradlew build` here.

```
Picked up _JAVA_OPTIONS: -Dawt.useSystemAAFontSettings=on -Dswing.aatext=true
This mapping 'snapshot_20180213' was designed for MC 1.12! Use at your own peril.
#################################################
         ForgeGradle 2.3-SNAPSHOT-ee3e3df        
  https://github.com/MinecraftForge/ForgeGradle  
#################################################
                 Powered by MCP                  
             http://modcoderpack.com             
     by: Searge, ProfMobius, R4wk, ZeuX          
     Fesh0r, IngisKahn, bspkrs, LexManos         
#################################################
Found AccessTransformer: oc_at.cfg
:extractMcpMappings
:deobfCompileDeobfDepTask0 SKIPPED
:deobfCompileDeobfDepTask1 SKIPPED
:deobfCompileDeobfDepTask2 SKIPPED
:deobfCompileDeobfDepTask3 SKIPPED
:deobfCompileDeobfDepTask4 SKIPPED
:deobfCompileDeobfDepTask5 SKIPPED
:deobfCompileDeobfDepTask6 SKIPPED
:deobfCompileRemapDepSourcesTask10 SKIPPED
:deobfCompileRemapDepSourcesTask7 SKIPPED
:deobfCompileRemapDepSourcesTask8 SKIPPED
:deobfCompileRemapDepSourcesTask9 SKIPPED
:deobfCompileDummyTask
:getVersionJson
:extractUserdev UP-TO-DATE
:downloadClient SKIPPED
:downloadServer SKIPPED
:splitServerJar SKIPPED
:mergeJars SKIPPED
:applyBinaryPatches SKIPPED
:deobfProvidedDummyTask
:extractDependencyATs SKIPPED
:extractMcpData SKIPPED
:genSrgs SKIPPED
:deobfMcMCP SKIPPED
:sourceApiJava UP-TO-DATE
:compileApiJava UP-TO-DATE
:sourceApiScala UP-TO-DATE
:compileApiScala UP-TO-DATE
:processApiResources UP-TO-DATE
:apiClasses UP-TO-DATE
:sourceMainJava UP-TO-DATE
:compileJava
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':compile'.
> Could not resolve :ExtraCells:api-1.12.2-2.5.11a48.
  Required by:
      li.cil.oc:OpenComputers:MC1.12.2-1.7.2+504c1c2f8
   > Could not resolve :ExtraCells:api-1.12.2-2.5.11a48.
      > Could not get resource 'http://addons-origin.cursecdn.com/files/2500/596/ExtraCells-api-1.12.2-2.5.11a48.xml'.
         > Could not GET 'http://addons-origin.cursecdn.com/files/2500/596/ExtraCells-api-1.12.2-2.5.11a48.xml'. Received status code 403 from server: Forbidden

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.


BUILD FAILED

Total time: 32.949 secs
```

--- a/build.gradle
+++ b/build.gradle
@@ -74,10 +74,7 @@ repositories {
     maven { url "http://tehnut.info/maven" }
 	maven { url "http://maven.thiakil.com" }
     maven { url "http://cc.crzd.me/maven/" }
-	ivy {
-        name 'ExtraCells'
-        artifactPattern "http://addons-origin.cursecdn.com/files/${config.extracells.cf}/[module]-[revision].[ext]"
-    }
+	maven { url "https://minecraft.curseforge.com/api/maven/" }
 }
 
 configurations {
@@ -101,9 +98,9 @@ dependencies {
 	provided ("appeng:appliedenergistics2:${config.ae2.version}:api") {
 		transitive = false
 	}
-	provided name: 'ExtraCells', version: config.extracells.version, ext: 'jar'
+	provided "extracells2:ExtraCells-api:${config.extracells.version}"
 	
-	provided (name: 'Mekanism', version: config.mekanism.version, classifier: 'api') {
+	provided("mekanism:Mekanism:${config.mekanism.version}:api") {
 		transitive = false
 	}
 

--- a/build.properties
+++ b/build.properties
@@ -9,13 +9,13 @@ mod.version=1.7.2
 ae2.version=rv5-stable-4
 cc.version=1.80pr1-build0
 extracells.cf=2500/596
-extracells.version=api-1.12.2-2.5.3a25
+extracells.version=1.12.2:2.5.3a25
 forestry.version=5.5.0.157
 hwyla.version=1.8.20-B35_1.12
 ic2.version=2.8.26-ex112
 jei.version=4.8.5.151
 mcmp.version=2.2.2_38
-mekanism.version=1.12.1-9.4.1.326
+mekanism.version=1.12.2-9.4.3.330
 tis3d.version=1.3.0.12
 
 maven.url=file:///home/www/maven.cil.li/web

